lua/nvim-treesitter/textobjects/move.lua

```diff
@@ -42,20 +42,31 @@ local function move(query_string, forward, start, bufnr)
     end
   end
 
-  local match = queries.find_best_match(bufnr, query_string, 'textobjects', filter_function, scoring_function)
-  local config = configs.get_module('textobjects.move')
+  local match =
+    queries.find_best_match(bufnr, query_string, query_group or "textobjects", filter_function, scoring_function)
```

Comment:
@synek @tom-anders I would just recommend to do the same for `folds`. Just have `"folds"` and `@fold` be the default but make it and option to change it to something else.